### PR TITLE
Add acceptance test for #803 et #804

### PR DESCRIPTION
**Pull request by pallix**

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/tests/acceptance/RateSpec.js [hunk 1]
```diff
@@ -34,4 +34,19 @@ describe("ratings", function() {
...
+        var page = new EmbeddedCommentsPage("c2").get();
+        var comment = page.createComment("c4");
+        var rate = page.getRateWidget(comment);
+
+        rate.element(by.css(".rate-pro")).click();
+
+        var changedComment = page.editComment(comment, ["c4 - edited"]);
+        var rate2 = page.getRateWidget(changedComment);
+        expect(rate2.element(by.css(".rate-difference")).getText()).toEqual("+1");
+
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/tests/acceptance/RateSpec.js]: Gardening: I would prefer no newline here. But that's not really important.

**Comment by xi** [comment]
Both tests fail as expected. They should be marked as `xit` (pending) so we can merge.

**Comment by pallix** [comment]
Is the test for issue #804 failing for you? It works for me locally

**Comment by xi** [comment]
I will check again

**Comment by xi** [comment]
In any case tests on travis should not fail. So in either case it must be marked `xit`.

**Comment by xi** [comment]
The test for #804 is failing for me when running `bin/protractor etc/protractorConf.js`

**Pull request merged**